app/src/main/java/org/p2p/wallet/auth/interactor/UserSignUpInteractor.kt

```diff
@@ -56,7 +56,10 @@ class UserSignUpInteractor(
 
     private suspend fun generateDeviceAndThirdShare(): Web3AuthSignUpResponse {
         val torusKey = signUpFlowDataRepository.torusKey
-            ?: error("Torus key is null")
-        return web3AuthApi.triggerSilentSignUp(torusKey)
+        if (torusKey.isNullOrEmpty()) {
+            error("Torus key is null")
```

Comment:
throw error("Torus key is null")?

Reply:
public inline fun error(message: Any): Nothing = throw IllegalStateException(message.toString())